Projects/Forever/Sources/DiscountCodeSection.swift

```diff
@@ -57,15 +53,30 @@ struct DiscountCodeSectionView: View {
         .presentableStoreLensAnimation(.spring())
         .sectionContainerStyle(.transparent)
     }
+
+    private func shareCode(code: String) {
```

Comment:
@sladan-hedvig not sure if we want to place this elsewhere?

Reply:
Maybe move it to the 
ForeverNavigationViewModel